oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1012,8 +1013,9 @@ void pldm::responder::oem_ibm_platform::Handler::setHostEffecterState(
     }
     else
     {
-        error("Invalid entity type received. Entity Type = {ENTITY_TYP}",
-              "ENTITY_TYP", entityTypeReceived);
+        error(
+            "Invalid entity type received by function setHostEffecterState. Entity Type = {ENTITY_TYP}",
```

Comment:
no need to write function name inside trace "setHostEffecterState".

Reply:
there are three places where we have same error message being printed due to which we took some extra time to debug so based on discussion with @Pavithrab7 this change was made.

Reply:
I have removed the entity type written in message but the other thing of using function name in lg2, have you seen any instance? I am aware of this thing working for std::cout but lg2 I am doubtful.

Reply:
will be omitting the file names.